src/NHibernate/Transaction/AdoNetWithDistributedTransactionFactory.cs

```diff
@@ -1,3 +1,4 @@
+#if !NETSTANDARD2_0
```

Comment:
@hazzik, so what do you suggest? From what I gather, there is a very narrow scenario that may work, but lots of NHibernate's tests fail with this factory.

I have never used distributed transactions, but at this point I would prefer to say that they are only supported in the 4.6.1 framework version, not in .net core. Mainly because I don't know what it will take for distributed transactions to work.